build.sbt

```diff
@@ -125,8 +139,8 @@ lazy val core = (project in file("modules/core"))
   .settings(
     name := "tesselation-core",
     scalacOptions ++= List("-Ymacro-annotations", "-Yrangepos", "-Wconf:cat=unused:info", "-language:reflectiveCalls"),
-    scalafmtOnCompile := true,
-    scalafixOnCompile := true,
+    scalafmtOnCompile := false,
+    scalafixOnCompile := false,
```

Comment:
What was the reason for this change? Rules were too restrictive?

Reply:
It was changing the code just before compiling it, leading to non-compilable code.

Reply:
bt actually that's wrong change. I will revert it